mix.exs

```diff
@@ -65,4 +75,24 @@ defmodule Wallaby.Mixfile do
       main: "readme",
     ]
   end
+
+  defp test_paths(driver) when driver in @drivers, do: ["integration_test/#{driver}"]
+  defp test_paths(_), do: ["test"]
+
+  defp test_drivers(args) do
+    for driver <- @drivers, do: run_integration_test(driver, args)
+  end
+
+  defp run_integration_test(driver, args) do
+    args = if IO.ANSI.enabled?, do: ["--color"|args], else: ["--no-color"|args]
+
+    IO.puts "==> Running tests for WALLABY_DRIVER=#{driver} mix test"
+    {_, res} = System.cmd "mix", ["test"|args],
+                          into: IO.binstream(:stdio, :line),
+                          env: [{"WALLABY_DRIVER", driver}]
+
+    if res > 0 do
+      System.at_exit(fn _ -> exit({:shutdown, 1}) end)
+    end
```

Comment:
This is the first time I've done this type of integration test setup, and I'll have to admin I copied and pasted from Ecto. 😅 I think the reason it's so manual is because it's adjusting environment variables, which then adjusts the `test_paths:` setting in the project config. We could probably find another way of getting mix to behave the way we want, but I think following Ecto's example isn't a bad for a first try.